packages/astro/test/static-build.test.js

```diff
@@ -96,6 +96,9 @@ describe('Static build', () => {
 			for (const link of links) {
 				const href = $(link).attr('href');
 
+				// The imported .scss file should include the base subpath in the href
+				expect(href.startsWith('/subpath/')).to.be.true;
```

Comment:
This test had a coverage bug in it that ignored the fact that "/subpath/" wasn't being used at all before this PR

This makes sure the test fixture's ESM import for a `.scss` file gets prepended with the subpath by Vite's build